core/src/main/scala/chisel3/internal/MonoConnect.scala

```diff
@@ -178,6 +188,35 @@ private[chisel3] object MonoConnect {
       case (sink, source) => throw MismatchedException(sink.toString, source.toString)
     }
 
+  /** Check whether two [[Vec]]s can be bulk connected (<-) in FIRRTL. */
+  private[chisel3] def canBulkConnectVecs(sink_v: Vec[Data @unchecked],
```

Comment:
This function and `canBulkConnectRecords` are also used in `BiConnect`. Is there a better place I should move these out to, or can they just live in `MonoConnect`?